spec/models/manageiq/providers/nuage/vsd_client/vcr_specs/vsd_client_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'json'
```

Comment:
Remove `json` dependency as @miha-plesko suggested that we are not interested in JSON parsing at all.